eo-maven-plugin/src/main/java/org/eolang/maven/Home.java

```diff
@@ -197,6 +199,10 @@ public static Path clean(final Path path) {
      * @return Absolute path
      */
     private Path absolute(final Path path) {
+        if (path.isAbsolute()) {
+            throw new IllegalArgumentException(format("Absolute path %s is prohibit.", path));
```

Comment:
@cosmincalinescu91 I believe more meaningful would be: "Absolute paths are prohibited: %s"